api/proto/info/info.proto

```diff
@@ -504,9 +506,44 @@ message ZInfoNetworkInstance {
     ZInfoVpn vinfo = 31;
     ZInfoLisp linfo = 32;
   }
+  UsageInfo usage = 33;
   repeated ErrorInfo networkErr = 40;
 }
 
+message UsageInfo {
+  google.protobuf.Timestamp createTime = 1;
+  uint32 refCount = 2;
+  google.protobuf.Timestamp lastUseTime = 3; // When refCount dropped to zero
+}
+
+message VolumeResources {
+  uint64 maxSizeBytes = 1; // From config
+  uint64 curSizeBytes = 2; // Current disk usage
+}
+
+message ZInfoVolume {
+  string uuid = 1;
```

Comment:
It might be useful to have a version too? Device can reflect back which version of the volume is being used - assuming we allow changing of volumes?

Reply:
@kalyan-nidumolu given that version is now gone from the config side we don't need to add it on the info side?